src/postToAPI.js

```diff
@@ -0,0 +1,12 @@
+async function postToAPI(score) {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3IGjFzzBEY0MX1krdUtm/scores/', {
+    method: 'POST',
+    body: JSON.stringify(score),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json());
+}
```

Comment:
**Changes Required**
- [ ] The use of `arrow function` is preferred to the `function` keyword. Let us refactor this into an ES6 arrow function. You can read more [here](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why you should prefer ES6 arrow functions to the `function keyword`.
- [ ] Nice try using the `async await` syntactic sugar to wrap your asynchronous code. However, you're mixing it with the `then` keyword. Let us remove the `then` keyword and just use the pure `async await`.